1.0/cxx/Dockerfile

```diff
@@ -39,12 +39,16 @@ ENV GRPC_RELEASE_TAG v1.0.0
 
 RUN git clone -b ${GRPC_RELEASE_TAG} https://github.com/grpc/grpc /var/local/git/grpc
 
+#install protoc
+RUN git clone https://github.com/google/protobuf.git 
+RUN cd protobuf && \
+    ./autogen.sh && ./configure && \
+    make && make check && make install && \
+    ldconfig && \
+    make && make install && make clean
```

Comment:
Why are you calling `make` and `make install` twice? it should be `make clean` after ldconfig

Reply:
thanks, you are right, it needs a minor change.